packages/components/src/checkbox/tests/chromatic/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,205 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { ErrorMessage } from "../../../errorMessage/index.ts";
+import { HelperMessage } from "../../../helperMessage/index.ts";
+import { Label } from "../../../Label/index.ts";
+import { Inline, Stack } from "../../../layout/index.ts";
+import { Text } from "../../../Text/index.ts";
+import { Checkbox } from "../../src/Checkbox.tsx";
+import { CheckboxField } from "../../src/CheckboxField.tsx";
+import { CheckboxGroup } from "../../src/CheckboxGroup.tsx";
+import { CheckboxList } from "../../src/CheckboxList.tsx";
+
+const meta = {
+    title: "Components/Checkbox/CheckboxGroup",
+    component: CheckboxGroup
+} satisfies Meta<typeof CheckboxGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Vertical = {
```

Comment:
Can you add an example with bordered + description, like in this example: 

https://6482172cea3f45dd1bb6ef3f-fhotrvaekq.chromatic.com/?path=/docs/docs-checkbox-checkboxgroup--docs#bordered